src/@newrelic/gatsby-theme-newrelic/icons/feather.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import featherIcons from '@newrelic/gatsby-theme-newrelic/src/icons/feather';
+
+export default {
+  ...featherIcons,
+  'external-link': (
```

Comment:
do we want to add this icon to the theme or is it more of a one-off for the dev site?

Reply:
I was wondering the same thing. I suspect that was the intention (thus the file name).

This seems like something that would be helpful across a few sites. I'll make another PR / release for this shortly.